guides/index_lifecycle.md

```diff
@@ -4,7 +4,7 @@ This guide covers OpenSearch Golang Client API actions for Index Lifecycle. You'
 
 ## Setup
 
-In this guide, we will need an OpenSearch cluster with more than one node. Let's use the sample [docker-compose.yml](https://opensearch.org/samples/docker-compose.yml) to start a cluster with two nodes. The cluster's API will be available at `localhost:9200` with basic authentication enabled with default username and password of `admin:admin`.
+In this guide, we will need an OpenSearch cluster with more than one node. Let's use the sample [docker-compose.yml](https://opensearch.org/samples/docker-compose.yml) to start a cluster with two nodes. The cluster's API will be available at `localhost:9200` with basic authentication enabled with default username and password of `admin:< Admin password >`.
```

Comment:
Lowercase Admin since the username is admin?

Reply:
Done in [d6f5ec4](https://github.com/opensearch-project/opensearch-go/pull/449/commits/d6f5ec4a64cd1ac36bfa017eacde24de127673f8)